package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prebid.js",
-  "version": "3.18.0-pre",
+  "version": "3.20.0-pre",
```

Comment:
you should not modify `package-lock.json` as part of the adapter submission PR, please revert changes you made to this file.